tests/test_food_db_new_realistic_coverage.py

```
@@ -98,21 +100,17 @@ def test_food_search_edge_cases_realistic(self):
             for term in search_terms:
                 try:
                     results = search_foods(term)
-                    # Should handle all cases gracefully
-                    assert isinstance(results, (list, dict, type(None)))
                 except Exception:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)